man/ocitools.1.md

```diff
@@ -15,11 +15,26 @@ ocitools is a collection of tools for working with the [OCI runtime specificatio
 
 # OPTIONS
 **--help**
-  Print usage statement
+  Print usage statement.
 
 **-v**, **--version**
   Print version information.
 
+**--log-level**
+  Log level (panic, fatal, error, warn, info, or debug) (default: "error").
+
+**--host-specific**
+  Generate host-specific configs or do host-specific validations.
+
+  By default, generator addes process capabilities into the config without
+  checking whether it is supported on the current host. With this flag,
+  generator will first check whether a process capability is supported on the
+  current host, and only add it into the config if it passes the checking.
```

Comment:
I recommend dropping this last sentence, since it will go stale as we add more host-specific stuff.

Alternatively, you could keep it as a “for example”, in which case we are clearly not aspiring towards completely documenting all host-specific generation.  If you go this route, I'd restore the Linux/Windows example to the validation paragraph below.